### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by J2TEAM**

### CODE REVIEW

File: js/configs.js [hunk 1]
```diff
@@ -164,30 +164,30 @@ class Main extends Component {
...
+            const checked = e.target.checked;
             if (checked === defaultValue) {
               delete userConfigs[group][props.config];
             } else {
               userConfigs[group][props.config] = checked;
             }
 
             this.setState({
-              userConfigs: userConfigs,
+              userConfigs: userConfigs
```
Comment (redphx) [inline js/configs.js]: @J2TeaM please [keep all trailing commas](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8)

File: js/content-script.js [hunk 2]
```diff
@@ -505,9 +505,9 @@ function overrideRequireJs(b, extensionId) {
   b.sendFbConfigs = function() {
     chrome.runtime.sendMessage(extensionId, {
       cmd: 'manageConfigs',
-      fbConfigs: fbConfigs,
+      fbConfigs: fbConfigs
```
Comment (redphx) [inline js/content-script.js]: @J2TeaM don't remove this too

File: js/content-script.js [hunk 3]
```diff
@@ -162,7 +162,7 @@ function overrideRequireJs(b, extensionId) {
           if (p) {
             var jb = r(),
               kb = 0;
-            if (ib != null) kb = hb.stopTiming(ib);
+            if (ib !== null) kb = hb.stopTiming(ib);
```
Comment (redphx) [inline js/content-script.js]: @J2TeaM I want to keep all the code from line 21 to line 457 beacause they're from Fb's code.

**Review by redphx — CHANGES_REQUESTED** [review]
@J2TeaM thanks for the PR. Can you update the code? Thanks.

File: js/content-script.js (hunk 3 above)
Comment (Daniel15) [inline js/content-script.js]: This change would make the code behave differently - `!= null` matches both `null` and `undefined`, whereas `!== null` only matches `null`.

Comment (redphx) [inline js/content-script.js]: @Daniel15 that's why I don't want to touch the original code

**Comment by J2TeaM** [comment]
@redphx Updated!

File: js/configs.js (hunk 1 above)
Comment (J2TeaM) [inline js/configs.js]: @redphx thanks! I learned something new today 👍

**Comment by redphx** [comment]
@J2TeaM thanks!

**Pull request merged**